### PR TITLE
DOC: Replace broken links by webarchive equivalent links

### LINKED ISSUE #3351: DOC: Broken link to Gabor tutorial
## Description
On this page:

http://scikit-image.org/docs/dev/api/skimage.filters.html#skimage.filters.gabor_kernel

the link to the Gabor tutorial:

http://mplab.ucsd.edu/tutorials/gabor.pdf

is broken.

### PR DESCRIPTION
**Pull request by sciunto**
## Description

Fixes #3351

## Checklist
[It's fine to submit PRs which are a work in progress! But before they are merged, all PRs should provide:]
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests

[For detailed information on these and other aspects see [scikit-image contribution guidelines](https://scikit-image.org/docs/dev/contribute.html)]

## References
[If this is a bug-fix or enhancement, it closes issue # ]
[If this is a new feature, it implements the following paper: ]

## For reviewers

(Don't remove the checklist below.)

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

File: skimage/filters/_frangi.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def frangi(image, scale_range=(1, 10), scale_step=2, beta1=0.5, beta2=15,
            vessel enhancement filtering," In LNCS, vol. 1496, pages 130-137,
            Germany, 1998. Springer-Verlag.
     .. [2] Kroon, D.J.: Hessian based Frangi vesselness filter.
-    .. [3] http://mplab.ucsd.edu/tutorials/gabor.pdf.
+    .. [3] https://web.archive.org/web/20180127125930/http://mplab.ucsd.edu/tutorials/gabor.pdf
```
Comment (soupault) [inline skimage/filters/_frangi.py]: I'm wondering how reliable web.archive.org links are :)...

An alternative could be a ResearchGate version, uploaded by the author - https://www.researchgate.net/publication/265064863_Tutorial_on_Gabor_Filters .

Comment (sciunto) [reply skimage/filters/_frangi.py]: I think it is less reliable... webarchive exists since a while

Comment (jni) [reply skimage/filters/_frangi.py]: Oh yeah, no way should we rely on ResearchGate, which is being sued by Elsevier. (Wrongly, but it's not like wrong things don't happen in law and life. =P) Web Archive is about as reliable as Wikipedia I think: a very valuable public good that is unlikely to go away without a fight.

Comment (sciunto) [reply skimage/filters/_frangi.py]: @jni So, ready to be merged? :)

**Comment by emmanuelle** [comment]
Merging, thanks.

**Pull request merged**